### PR TITLE
feat(sbb-sticky-bar): allow overlapping to the following content

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/components/container/sticky-bar/sticky-bar.stories.ts [hunk 1]
```diff
@@ -165,11 +165,15 @@ const WithContentAfterTemplate = ({
     <sbb-sticky-bar
       color=${color !== 'unset' ? color : nothing}
       ?disable-animation=${disableAnimation}
+      style="--sbb-sticky-bar-bottom-overlapping-height: var(--sbb-spacing-responsive-l);"
     >
       ${actionGroup()}
     </sbb-sticky-bar>
   </sbb-container>
-  <sbb-container color="milk">
+  <sbb-container color=${containerColor} aria-hidden>
```
Comment (jeripeierSBB) [inline src/components/container/sticky-bar/sticky-bar.stories.ts:173]:
```suggestion
  <sbb-container color=${containerColor} aria-hidden="true">
```

**Review by MarioCastigliano — APPROVED** [review]
Everything looks fine 👍
Let's test on Windows and we should be fine

**Review by mcilurzo — APPROVED** [review]

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Review by MarioCastigliano — COMMENTED** [review]
Code's looking great, and all edge-cases (that I could think of 😅) are still working fine 👍

**Pull request merged**